js/utils/CommonUtils.js

```diff
@@ -107,7 +107,10 @@ define([
 	function renderLink(s, p, d) {
 		var valid = d.INVALID_REASON_CAPTION == 'Invalid' ? 'invalid' : '';
 		var linkClass = getConceptLinkClass(d);
-		return '<a class="' + valid + ' ' + linkClass + '" href=\"#/concept/' + d.CONCEPT_ID + '\">' + filterXSS(d.CONCEPT_NAME) + '</a>';
+		const name = filterXSS(d.CONCEPT_NAME);
```

Comment:
I'd just apply the xss filter in the case of display because applying the filterXSS is an expensive operation (and will cause the sort mode of render to be very slow).

Second thing:  do we need to XSS concept name?  I'd expect the values in the vocabulary's concept names to be safe from XSS...

Reply:
@chrisknoll I removed filterXSS from this function. I've also added some filtration in linked-cohort-list and added strict xssOptions config to strip all html tags.

Reply:
@chrisknoll I remove filterXSS from this function. I've also added some filtration to linked-cohort-list with some strict xss options